src/lib/market-service/idle/idle.ts

```diff
@@ -37,16 +38,23 @@ export class IdleMarketService extends CoinGeckoMarketService implements MarketS
 
     this.baseUrl = 'https://api.idle.finance'
     this.providerUrls = providerUrls
-    this.idle = axios.create({
-      timeout: 10000,
-      baseURL: this.baseUrl,
-      headers: {
-        Authorization:
-          'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGllbnRJZCI6IkFwcDIiLCJpYXQiOjE2NzAyMzc1Mjd9.pf4YYdBf_Lf6P2_oKZ5r63UMd6R44p9h5ybPprtJmT4',
+
+    this.idle = setupCache(
+      axios.create({
+        timeout: 10000,
+        baseURL: this.baseUrl,
+        headers: {
+          Authorization:
+            'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGllbnRJZCI6IkFwcDIiLCJpYXQiOjE2NzAyMzc1Mjd9.pf4YYdBf_Lf6P2_oKZ5r63UMd6R44p9h5ybPprtJmT4',
+        },
+      }),
+      {
+        // 5 seconds
+        ttl: 5 * 1000,
+        interpretHeader: false,
```

Comment:
[interpretHeader](https://axios-cache-interceptor.js.org/config/request-specifics#cache-interpretheader): `If activated, when the response is received, the ttl property will be inferred from the requests headers. As described in the MDN docs and HTML specification.` 

Setting this to false as we set our own TTL. Equivalent to the previous `readHeaders: false`.